pkg/server/server.go

```diff
@@ -393,7 +395,8 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 			zap.Uint32("Threshold", s.dispatchThrottlingThreshold),
 		)
 
-		dispatchThrottlingCheckResolver := graph.NewDispatchThrottlingCheckResolver(dispatchThrottlingConfig)
+		s.dispatchThrottler = throttler.NewDispatchThrottler(dispatchThrottlingConfig)
```

Comment:
Nit: by saving this object as a server field, if you accidentally call `s.dispatchThrottler.Close()` (there is nothing stopping you from doing so) in `server.Close()`, you get a panic. I'm not sure why.

Reply:
Removed this param altogether